File: app/src/main/java/com/driveawayz/OTPScreen/response/NumberVerifyResponse.java
package com.driveawayz.OTPScreen.response;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class NumberVerifyResponse {

@SerializedName("sid")
@Expose
private String sid;
@SerializedName("serviceSid")
@Expose
private String serviceSid;
@SerializedName("accountSid")
@Expose
private String accountSid;
@SerializedName("to")
@Expose
private String to;
@SerializedName("channel")
@Expose
private String channel;
@SerializedName("status")
@Expose
private String status;
@SerializedName("valid")
@Expose
private Boolean valid;
@SerializedName("amount")
@Expose
private Object amount;
@SerializedName("payee")
@Expose
private Object payee;
@SerializedName("dateCreated")
@Expose
private String dateCreated;
@SerializedName("dateUpdated")
@Expose
private String dateUpdated;

public String getSid() {
return sid;
}

public void setSid(String sid) {
this.sid = sid;
}

public String getServiceSid() {
return serviceSid;
}

public void setServiceSid(String serviceSid) {
this.serviceSid = serviceSid;
}

public String getAccountSid() {
return accountSid;
}

public void setAccountSid(String accountSid) {
this.accountSid = accountSid;
}

public String getTo() {
return to;
}

public void setTo(String to) {
this.to = to;
}

public String getChannel() {
return channel;
}

public void setChannel(String channel) {
this.channel = channel;
}

public String getStatus() {
return status;
}

public void setStatus(String status) {
this.status = status;
}

public Boolean getValid() {
return valid;
}

public void setValid(Boolean valid) {
this.valid = valid;
}

public Object getAmount() {
return amount;
}

public void setAmount(Object amount) {
this.amount = amount;
}

public Object getPayee() {
return payee;
}

public void setPayee(Object payee) {
this.payee = payee;
}

public String getDateCreated() {
return dateCreated;
}

public void setDateCreated(String dateCreated) {
this.dateCreated = dateCreated;
}

public String getDateUpdated() {
return dateUpdated;
}

public void setDateUpdated(String dateUpdated) {
this.dateUpdated = dateUpdated;
}

}

File: app/src/main/java/com/driveawayz/OTPScreen/response/NumberVerifyResponse.kt
package com.driveawayz.OTPScreen.response

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class NumberVerifyResponse {
    @SerializedName("sid")
    @Expose
    private var sid: String? = null

    @SerializedName("serviceSid")
    @Expose
    private var serviceSid: String? = null

    @SerializedName("accountSid")
    @Expose
    private var accountSid: String? = null

    @SerializedName("to")
    @Expose
    private var to: String? = null

    @SerializedName("channel")
    @Expose
    private var channel: String? = null

    @SerializedName("status")
    @Expose
    private var status: String? = null

    @SerializedName("valid")
    @Expose
    private var valid: Boolean? = null

    @SerializedName("amount")
    @Expose
    private var amount: Any? = null

    @SerializedName("payee")
    @Expose
    private var payee: Any? = null

    @SerializedName("dateCreated")
    @Expose
    private var dateCreated: String? = null

    @SerializedName("dateUpdated")
    @Expose
    private var dateUpdated: String? = null

    fun getSid(): String? {
        return sid
    }

    fun setSid(sid: String?) {
        this.sid = sid
    }

    fun getServiceSid(): String? {
        return serviceSid
    }

    fun setServiceSid(serviceSid: String?) {
        this.serviceSid = serviceSid
    }

    fun getAccountSid(): String? {
        return accountSid
    }

    fun setAccountSid(accountSid: String?) {
        this.accountSid = accountSid
    }

    fun getTo(): String? {
        return to
    }

    fun setTo(to: String?) {
        this.to = to
    }

    fun getChannel(): String? {
        return channel
    }

    fun setChannel(channel: String?) {
        this.channel = channel
    }

    fun getStatus(): String? {
        return status
    }

    fun setStatus(status: String?) {
        this.status = status
    }

    fun getValid(): Boolean? {
        return valid
    }

    fun setValid(valid: Boolean?) {
        this.valid = valid
    }

    fun getAmount(): Any? {
        return amount
    }

    fun setAmount(amount: Any?) {
        this.amount = amount
    }

    fun getPayee(): Any? {
        return payee
    }

    fun setPayee(payee: Any?) {
        this.payee = payee
    }

    fun getDateCreated(): String? {
        return dateCreated
    }

    fun setDateCreated(dateCreated: String?) {
        this.dateCreated = dateCreated
    }

    fun getDateUpdated(): String? {
        return dateUpdated
    }

    fun setDateUpdated(dateUpdated: String?) {
        this.dateUpdated = dateUpdated
    }
}